### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by DavidBarker-EE**
Added local development setup and information about the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,78 @@
...
+# Form Builder
 
-**webpack-cli**
-Run `npm run build` to bundle application
+This project is a flexible and easy-to-use **Form Builder** designed to simplify the process of creating, customizing, and managing eligibility checkers. It leverages **Hapi.js** for the backend framework, with strong code standards enforced using **eslint** and **prettier**. The project is developed with **Node.js** (v20.x and above).
 
-## Local Setup
+## Features
+- Dynamic form creation
+- Field validation and customization
```
Comment (swdpcomputing) [inline README.md]: There's a few Americanisations in here. Are we fussed?

Comment (DavidBarker-EE) [reply README.md]: I'm fussed and appalled that I put that in there 😂

**Review by swdpcomputing — DISMISSED** [review]

**Review by swdpcomputing — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,22 +1,79 @@
...
+  npm run format
+  ```
+      
+**Run tests**
+
+```bash
+  npm test
+```
+
+This will run all the tests in the `test/` directory.
```
Comment (Andrew-Folga) [inline README.md]: Is this correct? It should run all tests even if not within /test folder. Unit tests won't be there.

**Comment by Andrew-Folga** [comment]
Can we add a table with env vars and explanation? Look in `config-map.yaml`:

```
  SERVICE_BUS_HOST: {{ .Values.container.messageQueueHost | quote }}
  SCORE_REQUEST_QUEUE_ADDRESS: {{ .Values.container.scoreRequestQueueAddress | quote }}
  SCORE_RESPONSE_QUEUE_ADDRESS: {{ .Values.container.scoreResponseQueueAddress | quote }}
  DESIRABILITY_SUBMITTED_TOPIC_ADDRESS: {{ quote .Values.container.desirabilitySubmittedTopicAddress }}
  NOTIFY_EMAIL_TEMPLATE: {{ .Values.container.notifyEmailTemplate | quote }}
```

File: README.md (hunk 2 above)
Comment (DavidBarker-EE) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,22 +1,79 @@
...
+Run `snyk auth` to authenticate your local machine with Snyk.
 
-Run `snyk auth` to authenticate your local machine
+## Project Structure
 
-### Tests
\ No newline at end of file
+- `src/` – Core application files (routes, handlers, etc.)
+- `public/` – Frontend assets (if applicable)
+- `test/` – Unit and integration tests
```
Comment (Andrew-Folga) [inline README.md]: This one also is not fully correct. I think we only have integration tests as it is now in `test/` folder.

**Review by Andrew-Folga — APPROVED** [review]
LGTM

**Pull request merged**